src/protocol/ipa_prf/boolean_ops/share_conversion_aby.rs

```diff
@@ -0,0 +1,361 @@
+#[cfg(all(test, unit_test))]
+use std::ops::Neg;
+
+use ipa_macros::Step;
+
+use crate::{
+    error::Error,
+    ff::{CustomArray, Field},
+    helpers::Role,
+    protocol::{
+        basics::PartialReveal, context::Context,
+        ipa_prf::boolean_ops::addition_low_com::integer_add, prss::SharedRandomness, RecordId,
+    },
+    secret_sharing::{replicated::semi_honest::AdditiveShare, WeakSharedValue},
+};
+#[cfg(all(test, unit_test))]
+use crate::{
+    ff::{boolean::Boolean, boolean_array::BA256, ec_prime_field::Fp25519, ArrayAccess},
+    secret_sharing::replicated::ReplicatedSecretSharing,
+};
+
+#[derive(Step)]
+pub(crate) enum Step {
+    GenerateSecretSharing,
+    IntegerAddBetweenMasks,
+    IntegerAddMaskToX,
+    RevealY,
+}
+
+/// share conversion from Boolean array of size n to integer mod 2^n
+/// See ABY3 (`https://eprint.iacr.org/2018/403.pdf`):
+///
+/// currently not used and also not tested
```

Comment:
hmmmm....

Reply:
lets remove this for now since we dont use it and I dont even test it